RFS/src/main/java/com/rfs/common/OpenSearchClient.java

```diff
@@ -257,20 +270,51 @@ public Optional<ObjectNode> getSnapshotStatus(
         }
     }
 
-    public Mono<BulkResponse> sendBulkRequest(String indexName, String body, IRfsContexts.IRequestContext context) {
+    Retry getBulkRetryStrategy() {
+        return Retry.backoff(10, Duration.ofSeconds(2)).maxBackoff(Duration.ofSeconds(60));
```

Comment:
Can we add a comment here why you chose 10? I'm thinking based on seeing the 10 minute cluster write locks?
Though in this case, since maxBackoff is 60s, the total delay won't actually be 10 minutes.

2 + 4 + 8 + 16 + 32 + 60 + 60 + 60 + 60 + 60 = 6 min 2 sec

Reply:
Upped this to 15 , where it will handle +10 minutes of an issue.